common/libs/messages.js

```diff
@@ -0,0 +1,7 @@
+// TODO - move this out of transaction; it's only for estimating gas costs
+export type TransactionWithoutGas = {|
+  to: string,
+  value: string | number,
+  data: string,
+  from: string
+|};
```

Comment:
Nothing suitable from `libs/transaction` for this?

Reply:
Nope :/